test/langtools/jdk/javadoc/doclet/testIndexInherited/pkg1/ClassB.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package pkg1;
+
+/**
+ * A public subclass in the same package as its superclass.
+ */
+public class ClassB implements ClassA {
+
+    /**
+     * The default constructor.
+     */
+    public ClassB() {
+    }
```

Comment:
One nit: here and elsewhere in this PR an explicit no-arg (parameterless, nullary, etc.) constructor is called _default_, which, technically speaking, it isn't. If a constructor is _default_, then it is implicit. Once you make it explicit, it is no longer _default_. A _default_ constructor is provided by the compiler, when there are no other constructors defined, see: JLS 8.8.9. Default Constructor.

Reply:
Fixed. Thank you, Pavel. I wondered that exact question while writing the comment, but I didn't bother to look it up. Thanks for taking the time to let me know. I changed the comments to "sole constructor."